zipkin.proto

```diff
@@ -212,3 +214,14 @@ message Annotation {
 message ListOfSpans {
   repeated Span spans = 1;
 }
+
+// Use this to report spans onto the gRPC transport. Implementations are asynchronous and may drop
```

Comment:
`SpanService allows reporting spans using gRPC, as opposed to the standard HTTP POST reporting endpoints. Implementations are...`

Reply:
thx